### PR TITLE
Add initial draft of phishing guide

### PR DESCRIPTION
**Pull request by claudijd**
Adding a more generic and somewhat non-intimidating guide to arm users with some basic tools for thwarting or detecting phishing attacks.

Probably something we can link to in security awareness training and phishing exercises as a training aid.

### CODE REVIEW

File: docs/guidelines/phishing.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+*The goal of this document is to help educate users about the risks of phishing attacks.  This document can be used both as a pro-active and a re-active aid to help reduce the likelihood of falling victim to a phishing attack.*
+
+# Background
+
+Phishing is an attack used by the computer hacking and fraud community to lure people to websites that they would normally use. They do this by creating e-mails that look very much like they are being sent by a legitimate company.
+
+Phishing emails often have the following characteristics:
+
+  - Phishing emails may ask you to click on a link in the e-mail, which would take you to a mock-up of the legitimate company's website where you are asked for your logon credentials or other sensitive or PII information.
```
Comment (gdestuynder) [inline docs/guidelines/phishing.md]: s/PII information/personal information/ (or just spell it out personally identifiable information)

File: docs/guidelines/phishing.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+  - Phishing emails can have a forged sender's address to mask an attacker's identity and make the email seem legitimate
```
Comment (gdestuynder) [inline docs/guidelines/phishing.md]: ~~How do I check this, as a user?~~
see below in bold

File: docs/guidelines/phishing.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+  - Attackers can also create hyperlinks inside of emails that are misleading
+  - Attackers will often use phishing emails with malicious attachments that are coerced to be opened or executed to compromise your system
```
Comment (gdestuynder) [inline docs/guidelines/phishing.md]: if im not wrong: the user is coerced, not the attachment

File: docs/guidelines/phishing.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
```
Comment (gdestuynder) [inline docs/guidelines/phishing.md]: ~~Would recommend having (parts of) an example email here, or inline with each issue, so that people that don't already know what is phishing or how it works can easily identify good/bad patterns~~

File: docs/guidelines/phishing.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+# Intuition
+
+By far and away, your own intuition is probably your best asset for easily detecting phishing attacks.  If an email seems out of place, unsolicited or asks you to take a weird action, stop and consider whether you are in a phishing scenario.  If yourself in this state, the below more technical information will be helpful in supporting those "weird feels" with evidence that would validate those concerns.   If at any time you get a feeling something isn't right, you are encouraged to investigate further, but if you feel out of your depth, it's encouraged to raise it with your information security team ([infosec@mozilla.com](mailto:infosec@mozilla.com) if at Mozilla) for further review and validation.
```
Comment (gdestuynder) [inline docs/guidelines/phishing.md]: "if yourself in this state"<=typo/grammar

note: if possible, break lines at 80 or 120 col. to help with review

Comment (gdestuynder) [inline docs/guidelines/phishing.md]: Because we attempt to write docs that are valid regardless of you being a Mozilla employee or not, would rather we write this such as:
"Reach out to your security team, for example [infosec@mozilla.com]"
Also, maybe it should be eus@mozilla.com?

File: docs/guidelines/phishing.md (hunk 4 above)
Comment (gdestuynder) [reply docs/guidelines/phishing.md]: **its actually down there in a different section, thus i'd in fact recommend just making this more concise or removing this entiere paragraph (no duplication)**

**Comment by claudijd** [comment]
Capturing some out of band feedback from Marshal...

1.) Consider having a "if at any point your concerned contact infosec" at the top (as opposed to repetitive mentions in each section)
2.) Provide more specific guidance relating to gmail (the mozilla email provider), rather than linking to an external source

File: docs/guidelines/phishing.md [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+
+One of the more common techniques with phishing emails is to abuse of from field content, this often looks like this...
+
+```
+From: Chris Beard (cbeard@mozilla.com) <abadguy@evil.com>
+To: You <you@example.com
+Subject: An Example Subject
+```
+
+If you observe this or some other behavior that suggests you've received a phishing email, it's important to raise it with your information security team ([infosec@mozilla.com](mailto:infosec@mozilla.com) if at Mozilla) for further review and attach the email headers for review.
```
Comment (gdestuynder) [inline docs/guidelines/phishing.md]: same comment as above regarding email

**Comment by claudijd** [comment]
I also re-read the document a realize I have a bunch of typos that I need to fix, consider the current state draft and that I'll go back and clean those up shortly.

File: docs/guidelines/phishing.md [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+
+A legitimate link for say mozilla.org would look like this when hovered...
+
+`https://mozilla.org/`
+
+However, an attacker may choose one of the following to try and trick you...
+
+`https://mozi11a.org/`
+
+If you observe this or some other behavior that suggests you've received a phishing email and have been cohersed into clicking a malicious link, it's important to raise it with your information security team ([infosec@mozilla.com](mailto:infosec@mozilla.com) if at Mozilla) for further review and attach the email headers (see above) and provide the example URL for further review.
```
Comment (gdestuynder) [inline docs/guidelines/phishing.md]: maybe we could just summarize this at the end of the doc instead of having  a paragraph each time asking people to email - such as "if you see any of the aforementioned email patterns, or have any doubt, please contact your security team" at the bottom or top of the doc

File: docs/guidelines/phishing.md [hunk 8]
```diff
@@ -0,0 +1,72 @@
...
+
+Phishing emails often contain attachments in the hope that you will click and run them and compromise your workstation.  If you need to open an attachment via email, make sure that you can confirm that the sender of that email is truely the sender and not a spoofed email made to look as if it was a trusted sender.  This can often be determined by examing the email headers (more details above) and generally whether that email was out of the blue from someone you don't talk to or simply is just not the norm.
+
+There are a number of malicious attachment types that are more dangerous that others, which include:
+
+- PDF Documents (very powerful, can have embedded malcious content)
+- MS Word/Excel Document (very powerful, especially in cases where Macros need to be enabled)
+- Bash/.exe Files (extremely dangerous)
+
+If you observe this or some other behavior that suggests someone is sending you unsolicited dangerous attachment types raise it with your information security team ([infosec@mozilla.com](mailto:infosec@mozilla.com) if at Mozilla) for further review and attach the email headers (see above) and provide the example attachment for further review.
```
Comment (gdestuynder) [inline docs/guidelines/phishing.md]: and the last one again :)

**Comment by claudijd** [comment]
Per kang: add a section for user opsec and add this to it.

**Comment by gdestuynder** [comment]
note: you might want to hold until you have at least 2 or 3 user opsec docs for this, but yeah :)

File: docs/guidelines/phishing.md [hunk 9]
```diff
@@ -0,0 +1,72 @@
+---
+layout: default
+resource: true
+categories: [Guidelines]
+title: Phishing
+description: A fraudulent practice of sending emails (or other communications) purporting to be from reputable companies in order to induce individuals to reveal personal information, such as passwords and credit card numbers.
+---
+
+*The goal of this document is to help educate users about the risks of phishing attacks.  This document can be used both as a pro-active and a re-active aid to help reduce the likelihood of falling victim to a phishing attack.*
```
[the remark this answers is not in the document]
Comment (caggle) [reply docs/guidelines/phishing.md]: I'd say it's the former, rather than educating.

[the remark this answers is not in the document]
Comment (caggle) [reply docs/guidelines/phishing.md]: It should be "reactive"

File: docs/guidelines/phishing.md (hunk 3 above)
Comment (caggle) [reply docs/guidelines/phishing.md]: Yeah, this was my mistake...
How about something like: "Attackers will often use phishing emails with malicious attachments, which if opened or executed, could compromise your system.

File: docs/guidelines/phishing.md (hunk 7 above)
Comment (caggle) [reply docs/guidelines/phishing.md]: I agree. Sounds like this is what Marshal is suggesting also.

File: docs/guidelines/phishing.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (claudijd) [reply docs/guidelines/phishing.md]: removed the second sentence.

[the remark this answers is not in the document]
Comment (claudijd) [reply docs/guidelines/phishing.md]: removed the second sentence

File: docs/guidelines/phishing.md (hunk 1 above)
Comment (claudijd) [reply docs/guidelines/phishing.md]: Removed, there was feedback that this section was redundant.

File: docs/guidelines/phishing.md (hunk 3 above)
Comment (claudijd) [reply docs/guidelines/phishing.md]: Removed the entire section

File: docs/guidelines/phishing.md (hunk 4 above)
Comment (claudijd) [reply docs/guidelines/phishing.md]: removed duplication.

File: docs/guidelines/phishing.md (hunk 5 above)
Comment (claudijd) [reply docs/guidelines/phishing.md]: Moved to infosec@, and put at the top in a single place.  also changed other sections to refer to gneric. "security team" to be non-mozilla specific

File: docs/guidelines/phishing.md (hunk 6 above)
Comment (claudijd) [reply docs/guidelines/phishing.md]: FIxed

File: docs/guidelines/phishing.md (hunk 8 above)
Comment (claudijd) [reply docs/guidelines/phishing.md]: Fixed

**Comment by claudijd** [comment]
I'm going to follow kang suggestion of breaking this into a user section once we have more than one.  for now, this can live in the existing header section.

**Comment by caggle** [comment]
Looks neat!

File: docs/guidelines/phishing.md [hunk 10]
```diff
@@ -0,0 +1,70 @@
...
+layout: default
+resource: true
+categories: [Guidelines]
+title: Phishing
+description: A fraudulent practice of sending emails (or other communications) purporting to be from reputable companies in order to induce individuals to reveal personal information, such as passwords and credit card numbers.
+---
+
+*The goal of this document is to help users figure out if their being phished and escalate accordingly.*
+
+If at any point you believe you are being targeted in a phishing campaign, it's important to raise those concerns with your security team. For example, if you are a mozilla employee or contributor, you would use [infosec@mozilla.com](mailto:infosec@mozilla.com).
```
Comment (gdestuynder) [inline docs/guidelines/phishing.md]: s/mozilla/Mozilla/

Comment (gdestuynder) [reply docs/guidelines/phishing.md]: s/you would use../please contact https://infosec.mozilla.org/#contact/

File: docs/guidelines/phishing.md [hunk 11]
```diff
@@ -0,0 +1,70 @@
...
+
+```
+From: Chris Beard (cbeard@mozilla.com) <abadguy@evil.com>
+To: You <you@example.com>
+Subject: An Example Subject
+```
+
+# Links
+
+Links in email addresses can be misleading when crafted in HTML, much like a link on any webpage.  It's important to hover over any links provided in emails before clicking them to ensure they take you to the correct domain.  This hover action will present you with a preview of the URL in the bottom of your email client.
```
Comment (gdestuynder) [inline docs/guidelines/phishing.md]: I would reword that (its not very clear what a link in email is):
HTML documents can contain links to email addresses such as `mailto:...`

(its not clear otherwise)

File: docs/guidelines/phishing.md [hunk 12]
```diff
@@ -0,0 +1,70 @@
...
+
+A common tactic for detecting malicious URLs in email is simply making sure the link is to the right domain. Phishing attacks will often leverage common looking domains with one character subsituted in the hopes of getting you to trust them.
+
+A legitimate link for mozilla.org would look like this when hovered...
+
+`https://mozilla.org/`
+
+However, an attacker may replace the 'l' with a numerical 1 to try and trick you...
```
Comment (gdestuynder) [inline docs/guidelines/phishing.md]: s/'l'/`l`/ (highlights it)

**Comment by gdestuynder** [comment]
%claudijd | kang: lgtm, ship it!
